### ISSUE TITLE
Remove Case 7.9.3

### ISSUE THREAD

**Issue by kelunik** [crossbario/autobahn-testsuite#79]
**Case Description**

Send close with invalid close code 1004

**Case Expectation**

Clean close with protocol error code or drop TCP

According to https://tools.ietf.org/html/rfc6455#section-7.4.1 this code is reserved, but unlike 1005 or 1006, there's no language that forbids using the code, it therefore doesn't make sense to me to expect a 1002 or empty close reason. We currently send 1000 instead, which makes the test fail.

**Comment by nhooyr** [comment]
Agreed, if in the future it has a real meaning, then all existing clients will throw needless errors when encountering it. It makes more sense for them to just echo it as normal and then let application code deal with its meaning.

**Comment by nhooyr** [comment]
Actually, if its reserved, it could become a code that isn't allowed on the wire, it's pretty much undefined so it does make sense to preclude it from going on the wire.
